acceptance-tests/src/main/java/io/blueocean/ath/pages/blue/GithubCreationPage.java

```diff
@@ -0,0 +1,111 @@
+package io.blueocean.ath.pages.blue;
+
+import io.blueocean.ath.WaitUtil;
+import io.blueocean.ath.api.classic.ClassicJobApi;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.IOException;
+
+@Singleton
+public class GithubCreationPage {
+    private Logger logger = Logger.getLogger(GithubCreationPage.class);
+
+    @Inject
+    public GithubCreationPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(xpath = "//span[text()='Github']")
```

Comment:
This one jumped out at me because the proper casing is actually "GitHub"... which we'll invariably update at some point and break this test :P  I think we should just add a className to the button like `scm-button-github`